week1/chileshe.js

```diff
@@ -0,0 +1,54 @@
+"use strict";
+
+const http = require("http");
+
+const server = http.createServer((req, res) => {
+    console.log("Creating server");
+});
+
+server.on("connection", () => {
+    console.log("Connected")
+});
+
+
+const PORT = 8080;
+
+server.listen(PORT, () => {
+    console.log("Listening on port", PORT);
+});
+
+
+let state = 10;
+
+server.on("request", (req, res) => {
+
+    res.setHeader("content-type", "text/html");
+
+    if (req.url === "/state") {
```

Comment:
`if/elseif/else` works okay here, but it's not ideal. Can you think of another way to handle all the different cases?